spec/unit/api_client_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe Typekitter::ApiClient do
+  describe '#kits_index' do
+    context 'request to typekit api is successful' do
+      it 'returns a parsed string of typekit kits' do
+        json_response = "{\"foo\":[\"success\"]}"
+        parsed_kits = JSON.parse(json_response)
+        fake_token = double("token", value: 'fake token 1234')
+        stub_request(:get, "https://typekit.com/api/v1/json/kits").
+         with(:headers => {
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indenting an expression spanning multiple lines.